.github/ISSUE_TEMPLATE/component_validation_checklist.md

```diff
@@ -0,0 +1,72 @@
+# Component validation checklist
+
+Here is a checklist of items that need to be completed before a component or pattern can be marked as **Ready** in the CMS Design System.
+
+## Documentation is complete
+
+**What to check:**
+- [ ] Description of component or pattern
+- [ ] HTML code snippet example(s)
+- [ ] React code snippet example(s)
+- [ ] All react properties are explained
+- [ ] General guidance or best practices
+- [ ] Usability - when to use
+- [ ] Usability - when to consider something else
+- [ ] Accessibility considerations
+
+## Validate HTML
```

Comment:
I'd like to call this section Validate Markup. I think we can use the bullet points to describe what will be validated in each step.